src/App.jsx

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={4}
+    />
```

Comment:
```suggestion
    <Sum a={2} b={3}/>
```
It is better to make it in one line if there are < 3 props.
Also, there should be a sum of 2 and 3.
Fix formatting for all cases

Reply:
https://Artem193.github.io/react_sum/